### ISSUE TITLE
Edge Nodes Failing License Check

### ISSUE THREAD

**Issue by mcaulifn** [jfrog/terraform-provider-artifactory#523]
**Describe the bug**
Edge nodes are failing the license check. `check_license = false` is being required to work around issue.

**Requirements for and issue**
- [ ] A description of the bug
- [ ] A fully functioning terraform snippet that can be copy&pasted (no outside files or ENV vars unless that's part of the issue). **If this is not supplied, this issue will likely be closed without any effort expended.**
- [ ] Your version of artifactory (you can `curl` it at `$host/artifactory/api/system/version`
- [ ] Your version of terraform
- [ ] Your version of terraform provider

**Expected behavior**
Edge nodes should not need to have the license check bypassed as they are included in the necessary license levels.

**Additional context**

curl -H "Authorization: Bearer $TOKEN"  https://xxxx.jfrog.io/artifactory/api/system/license
{
  "type" : "Edge",
  "validThrough" : "N/R",
  "licensedTo" : "Artifactory Online Dedicated"
}
Terraform v1.2.5

provider registry.terraform.io/jfrog/artifactory v6.11.2
provider registry.terraform.io/jfrog/project v1.1.3

**Comment by mcaulifn** [comment]
Same issue as #284

**Comment by alexhung** [comment]
@mcaulifn I think the issue originates from the [Project provider](https://github.com/jfrog/terraform-provider-project/blob/master/pkg/project/provider.go#L88). I'm almost 100% sure the Artifactory provider works with the 'Edge' license.

Is it possible for you to test it with Project provider only?

**Comment by mcaulifn** [comment]
You are correct, it is `projects`. Can you move it or should I create a new one?

**Comment by alexhung** [comment]
@mcaulifn I can move the ticket
